constants.js

```diff
@@ -62,7 +62,7 @@ const checksWhitelist = {
     [editEvent]: [],
     [issuesLabelEvent]: []
   },
-  'oppia': {
+  oppia: {
```

Comment:
Unneeded change.

Reply:
Fixed. Disable lint for that line. [PTAL](https://github.com/oppia/oppiabot/pull/233#pullrequestreview-622700445)